Examples/TestInsets/main.js

```diff
@@ -0,0 +1,23 @@
+/* 
+
+And example to demonstrate how to use insets on Rects
+
+*/
+
+
+var r1 = new Rect({x:100, y:100, width:250, height:250})
+
+console.log(r1.minX+","+r1.minY+":"+r1.maxX+","+r1.maxY);
```

Comment:
Ooh, that reminds me: we should override `toString()` on these things. Filed #59.